src/dialogs/examples/basic/O365ConnectorCardActionsDialogV1.ts

```diff
@@ -0,0 +1,206 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as teams from "botbuilder-teams";
+
+export class O365ConnectorCardActionsDialogV1 extends TriggerActionDialog {
```

Comment:
Please rename either according to what these examples show, or drop the "V". The V1/V2/V3 name makes me think that there are several versions of actionable cards, and these are examples of each version.

Reply:
Done